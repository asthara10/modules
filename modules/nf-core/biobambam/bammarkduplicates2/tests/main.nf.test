nextflow_process {

    name "Test Process BIOBAMBAM_BAMMARKDUPLICATES2"
    script "../main.nf"
    process "BIOBAMBAM_BAMMARKDUPLICATES2"

    tag "modules"
    tag "modules_nfcore"
    tag "biobambam"
    tag "biobambam/bammarkduplicates2"

    test("test_biobambam_bammarkduplicates2") {


        when {
            process {
                """
                
                input[0] = [
        [ id:'test', single_end:false ], // meta map
        file(params.test_data['sarscov2']['illumina']['test_paired_end_bam'], checkIfExists: true) 
    ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
