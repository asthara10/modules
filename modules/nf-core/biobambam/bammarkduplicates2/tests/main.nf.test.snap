{
    "versions": {
        "content": [
            [
                
            ]
        ],
        "timestamp": "2023-11-24T14:38:12.027291"
    }
}